pkg/wfc3/calwf3/lib/dodqi.c

```diff
@@ -194,6 +211,14 @@ SingleGroup *x    io: image to be calibrated; DQ array written to in-place
 					sum_dq = DQPix (x->dq.data, i, j) | ATODSAT;
 					DQSetPix (x->dq.data, i, j, sum_dq); /* atod sat */
 				}
+
+				if (wf3->scalar_satflag == True) {
```

Comment:
Are lines 215-2212 to be executed after lines 210-213, or instead of?

Reply:
Lines 215-221 are executed after Lines 210-213.